Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -57,15 +57,31 @@ bool LogFileReader::readAll(const QString& fileName)
  * that the conversion from string to double is sucessful.*/
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
+    bool check = true;
+
     QStringList sections = line.split(BATDATA_DELIMITER);
 
     QString timeString = sections.at(0);
+
+    if (sections.size() != 3)
```

Comment:
Lets try to avoid 'magic numbers' as much as possible, better to declare a constant in the namespace above.